modules/widget-visibility/widget-conditions.php

```diff
@@ -235,6 +235,50 @@ public static function widget_conditions_options_echo( $major = '', $minor = ''
 				</optgroup>
 				<?php
 			break;
+
+			case 'single_post':
+				// Get Post object.
+				$post_types = array(
+					'post' => get_post_type_object( 'post' )
+				);
+
+				$custom_post_types = get_post_types(
+					array(
+						'public'   => true,
+						'_builtin' => false,
+					),
+					'objects'
+				);
+
+				// Add "Post" object to other post types.
+				$post_types = array_merge( $post_types, $custom_post_types );
+
+				foreach ( $post_types as $post_type ) {
+				?>
+					<optgroup label="<?php printf( esc_attr( '%s :', 'jetpack' ), $post_type->label ); ?>">
```

Comment:
Yeah, you are right again :)
I have removed the `printf` along with the colon.
And I have renamed the option to "Post" only. Not sure what is the better name for this option, because we will display different post type posts.